edgexfoundry/edgex-compose
# feat: Add new App Record & Replay app service

closes # 392







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instruction
run `make run ds-virtual as-record-replay` from compose builder
Verify App Record Replay bootstraps and configuration is in Consul
Send GET to localhost:59712/api/v3/ping
Verify 200 response
Send GET to https://localhost:8443/app-record-replay/api/v3/ping with token set for API gateway access
Verify 200 response
